var/spack/repos/builtin/packages/py-h5py/package.py

```diff
@@ -42,12 +43,13 @@ class PyH5py(PythonPackage):
     depends_on("python@3.7:", type=("build", "run"), when="@3.2:")
 
     # Build dependencies
-    depends_on("py-cython@0.23:", type="build", when="@:2")
-    depends_on("py-cython@0.29:", type=("build"), when="@3: ^python@:3.7")
-    depends_on("py-cython@0.29.14:", type=("build"), when="@3: ^python@3.8.0:3.8")
-    depends_on("py-cython@0.29.15:", type=("build"), when="@3: ^python@3.9.0:")
+    depends_on("py-cython@0.23:0", type="build", when="@:2")
+    depends_on("py-cython@0.29:0", type=("build"), when="@3: ^python@:3.7")
+    depends_on("py-cython@0.29.14:0", type=("build"), when="@3: ^python@3.8.0:3.8")
+    depends_on("py-cython@0.29.15:0", type=("build"), when="@3: ^python@3.9.0:")
```

Comment:
```suggestion
    depends_on("py-cython@0.29:0", type=("build"), when="@3:")
    depends_on("py-cython@0.29.14:0", type=("build"), when="@3:3.7 ^python@3.8")
    depends_on("py-cython@0.29.15:0", type=("build"), when="@3:3.7 ^python@3.9:")
```
This constraint no longer applies in  3.8.0

Reply:
Done